components/scream/src/physics/p3/atmosphere_microphysics_run.cpp

```diff
@@ -24,6 +24,10 @@ void P3Microphysics::run_impl (const int dt)
   workspace_mgr.reset_internals();
 
   // Run p3 main
+  get_field_out("micro_liq_ice_exchange").deep_copy(0.0);
```

Comment:
Luca, does this fix the issue we were seeing with restarts and these fields?

Reply:
Yes. At least it fixed it on this branch for the `model_restart` test.